test/karma-test-shim.js

```diff
@@ -22,17 +22,66 @@ function getPathsMap(dir) {
 }
 
 System.config({
+  baseURL: '/base/dist',
   packages: {
-    'base/dist/components': {
-      defaultExtension: false,
+    'components': {
       format: 'cjs',
-      map: getPathsMap('components')
+      map: getPathsMap('components'),
+      defaultExtension: 'js'
     },
-    'base/dist/core': {
-      defaultExtension: false,
+    'core': {
       format: 'cjs',
-      map: getPathsMap('core')
+      map: getPathsMap('core'),
+      defaultExtension: 'js'
     },
+    '@angular/core': {
```

Comment:
Nitpick: we could probably build this object with a function and an array of bundle names to save the copy-pasting. But probably outside the scope of this PR